personal/privacyidea/class_mfaAccount.inc

```diff
@@ -192,11 +192,70 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
-        // Always mark this as an account.
         if (!$this->is_account && $this->acl_is_writeable("")) {
             $this->is_account = true;
         }
 
+        // Act on account stat changes
+        $display = "";
+        if ($this->parent instanceof roletabs || $this->parent instanceof uniBonnAccountTabs) {
```

Comment:
This whole block is customer specific, please remove.

Reply:
Removed.